src/modules/launcher/Plugins/Microsoft.Plugin.Uri.UnitTests/UriHelper/ExtendedUriParserTests.cs

```diff
@@ -132,7 +132,7 @@ public class ExtendedUriParserTests
         // All following cases should be parsed as application URI
         [DataRow("mailto:", true, null, "mailto:")]
         [DataRow("mailto:/", false, null, null)]
-        [DataRow("mailto:example@mail.com", true, null, "mailto:example@mail.com")]
+        [DataRow("mailto:mail@example.com", true, null, "mailto:mail@example.com")]
```

Comment:
[rfc2606](https://www.rfc-editor.org/rfc/rfc2606.html) provides for certain domains for use in cases like this, whereas `mail.com` is a live domain (not, to my knowledge, owned by Microsoft).